bin/BedTools.py

```diff
@@ -1,10 +1,10 @@
 import sys
 import re
 import os
+import pybedtools
```

Comment:
This script could have a python3 shebang (`#!/usr/bin/env python3`) at the top, although it works without it. It should also have author information (Original script by James Torrance, rewritten by Damon-Lee Pointon and Yumi Sims)

Reply:
haven't done this, I will make a separate pr for the bedtools